File: Bike2Beans/Application/DTOs/RouteStopDto.cs
using Bike2Beans.Domain.Entities;

namespace Bike2Beans.Application.DTOs;

public record RouteStopDto(
    Guid Id,
    string PlaceId,
    string Name,
    string? Address,
    LocationTypeEnum LocationType,
    double Lat,
    double Lng
);

File: Bike2BeansUI/src/Data/RouteStopDto.ts
export enum RouteStopLocationType {
    Coffeeshop = 0,
    Landmark = 1,
    Other = 2,
}

export type RouteStopDto = {
    id: string;
    placeId: string;
    name: string;
    address: string;
    locationType: RouteStopLocationType;
    lat: number;
    lng: number;
};
